packages/tx/src/eip1559Transaction.ts

```diff
@@ -0,0 +1,353 @@
+import { BN, bnToHex, bnToRlp, ecrecover, keccak256, rlp, toBuffer } from 'ethereumjs-util'
+import { BaseTransaction } from './baseTransaction'
+import {
+  AccessList,
+  AccessListBuffer,
+  FeeMarketEIP1559TxData,
+  FeeMarketEIP1559ValuesArray,
+  JsonTx,
+  N_DIV_2,
+  TxOptions,
+} from './types'
+import { AccessLists } from './util'
+
+const TRANSACTION_TYPE = 2
+const TRANSACTION_TYPE_BUFFER = Buffer.from(TRANSACTION_TYPE.toString(16).padStart(2, '0'), 'hex')
+
+export default class FeeMarketEIP1559Transaction extends BaseTransaction<FeeMarketEIP1559Transaction> {
+  public readonly chainId: BN
+  public readonly accessList: AccessListBuffer
+  public readonly AccessListJSON: AccessList
+  public readonly maxInclusionFeePerGas: BN
+  public readonly maxFeePerGas: BN
+
+  /**
+   * EIP-2930 alias for `r`
+   */
+  get senderR() {
```

Comment:
is there a reason to continue to add these aliases? i think holger alternatively suggested to just move them to baseTransaction so they are usable by all.

Reply:
I have no strong opinion here, what we want to do in the end is see what other client use for field names and also start using those. Alternatively, we could also use the names as specified in the EIPs.